packages/core/src/libraryUtils.ts

```diff
@@ -98,6 +98,8 @@ export { fetchAuthSession } from './singleton/apis/internal/fetchAuthSession';
 export { AMPLIFY_SYMBOL } from './Hub';
 export { base64Decoder, base64Encoder } from './utils/convert';
 export { getCrypto } from './utils/globalHelpers';
+export { cryptoSecureRandomInt } from './utils/cryptoSecureRandomInt';
+export { WordArray };
```

Comment:
Nit, and remove import on line 3

```suggestion
export { WordArray } from './utils/WordArray';
```